### PR TITLE
added chef install POC

### PR DESCRIPTION
**Pull request by VulpesArtificem**
https://rackhd.atlassian.net/browse/RAC-4146

added a graph to install, configure, and run the chef client on a node. 
@pscharla @RackHD/corecommitters @panpan0000

### CODE REVIEW

**Comment by panpan0000** [comment]
:+0 . Thank you for your quick update @VulpesArtificem !
looking forward to your PR of updating doc repo :-)

**Comment by panpan0000** [comment]
@RackHD/corecommitters , if this PR can't be merged in 2.0.0
it will be schedule risk for Dell EMC World.

**Comment by pscharla** [comment]
jenkins: test this please

**Comment by pscharla** [comment]
@RackHD/corecommitters : please take a look at this one. Thanks!

**Comment by PengTian0** [comment]
BUILD [on-taskgraph #5](http://147.178.202.18//blue/organizations/jenkins/on-taskgraph/detail/on-taskgraph/5/pipeline) : FAILURE

**Comment by panpan0000** [comment]
test this please

File: lib/graphs/examples/install-chef-graph.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+// Copyright 2017, EMC, Inc.
```
Comment (anhou) [inline lib/graphs/examples/install-chef-graph.js]: please use the latest copyright header

Comment (pscharla) [inline lib/graphs/examples/install-chef-graph.js]: @VulpesArtificem : FYI, this is the new copyright message we should use:
Copyright © 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.

**Comment by pscharla** [comment]
@anhou : If you're good with this one now, please approve and merge. Thanks!

File: spec/lib/graphs/install-chef-graph-spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Copyright 2017, EMC, Inc.
```
Comment (anhou) [inline spec/lib/graphs/install-chef-graph-spec.js]: this also need to be updated 'Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.'

Comment (pscharla) [inline spec/lib/graphs/install-chef-graph-spec.js]: @VulpesArtificem : Need other file to have the same copyright. Thanks!

**Comment by pscharla** [comment]
jenkins: test this please

**Comment by pscharla** [comment]
@anhou : I think this one is OK to merge.

**Pull request merged**